sync.go

```diff
@@ -433,16 +361,51 @@ func (lw *LibWallet) RescanBlocks() error {
 		// Trigger sync completed callback.
 		// todo: probably best to have a dedicated rescan listener
 		// with callbacks for rescanStarted, rescanCompleted, rescanError and rescanCancel
-		for _, syncProgressListener := range lw.syncProgressListeners {
-			syncProgressListener.OnSyncCompleted()
-		}
+		// for _, syncProgressListener := range lw.syncProgressListeners {
```

Comment:
Was this commented out for test purposes?

Reply:
Commented out to be considered later. New design doesn't come with a rescan function and might consider removing it.